overlord/ifacestate/helpers.go

```diff
@@ -526,3 +527,17 @@ func snapsWithSecurityProfiles(st *state.State) ([]*snap.Info, error) {
 
 	return infos, nil
 }
+
+func resolveSnapIDToName(st *state.State, snapID string) (name string, err error) {
+	if snapID == "system" {
```

Comment:
Do we have a test that uses the system snapID? And if not, should we have one?

Reply:
TestPopulateFromSeedGadgetConnectHappy  exercises that path as well